deploy/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ if(WIN32)
     if(MINGW)
         set(_ZLIB_DLL_FILENAME "libzlib.dll")
     else()
-        set(_ZLIB_DLL_FILENAME "zlib.dll")
+        set(_ZLIB_DLL_FILENAME "zlib1.dll")
     endif()
-    install(FILES "${_ZLIB_DLL_PATH}/${_ZLIB_DLL_FILENAME}" DESTINATION ".")
+    install(FILES "${_ZLIB_DLL_PATH}/${_ZLIB_DLL_FILENAME}" DESTINATION "." RENAME "zlib.dll")
```

Comment:
With the above "fix" you should no longer need to rename the dll.

Reply:
Yep removed it as no more required. Thanks